packages/code-du-travail-data/indexing/cdtnIds.js

```diff
@@ -0,0 +1,49 @@
+import * as XXH from "xxhashjs";
+
+// never change this seed, as ids are regenerated
+// each time but must remain stable
+const H = XXH.h64(0x1e7f);
+
+// default id length, can be longer (up to 16)
+const maxIdLength = 10;
+
+// use xxhash to hash source+id
+const hashId = (source, id, idLength) =>
```

Comment:
we could concatenate source and id before calling  the function to make it a bit generic ?

Reply:
yeah, I just wanted to make it explicit that hashing will be done on both in order to avoid "inter source" collision